app/src/ui/app.tsx

```diff
@@ -583,6 +585,25 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClick() {
+    if (__DARWIN__) {
+      const actionOnDoubleClick: AppleActionOnDoubleClickPref = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
+      const mainWindow: Electron.BrowserWindow = remote.getCurrentWindow()
```

Comment:
We don't need the `Electron.BrowserWindow` declaration here - this can be inferred, unlike the previous variable.